snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml

```diff
@@ -51,6 +59,9 @@ spec:
           - kind: Secret
             name: snyk-monitor
             version: v1
+          - kind: PersistentVolumeClaim
+            name: snyk-monitor-pvc
```

Comment:
@ivanstanev Is this OK given that the pvc's name could be overridden by a Value?

Reply:
I am not sure, we could test it with a PVC named differently than this! But I think these fields here are going to be shown in the UI, perhaps not critical and won't block deployment